### PR TITLE
feat: add connection indicator

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR:
- add a connection indicator that tries to get `/status` from the backend

Screenshot 2024-03-22 at 16 39 24

### CODE REVIEW

File: src/components/APIChecker.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          />
+        </svg>
+        <Typography
+          sx={{ textShadow: 'currentColor 1px 0px 10px' }}
+          variant="caption"
+        >
+          Connected to Graasp backend
+        </Typography>
+      </Stack>
+    );
```
Comment (pyphilia) [inline src/components/APIChecker.tsx:44]: I wouldn't show this. It should be trivial that the backend is connected.

File: src/langs/en.json [hunk 2]
```diff
@@ -23,5 +23,9 @@
   "SIGN_UP_SAVE_ACTIONS_LABEL": "Enable Analytics",
   "SIGN_UP_SAVE_ACTIONS_TOOLTIP": "Coming Soon!",
   "SIGN_UP_SUCCESS_TITLE": "Welcome!",
-  "SWITCH_ACCOUNT_TEXT": "Switch to another account"
+  "SWITCH_ACCOUNT_TEXT": "Switch to another account",
+  "API_UNAVAILABLE_TITLE": "Server Connection Error",
+  "API_UNAVAILABLE_EXPLANATION": "The connection with the Graasp server seems to be broken.",
```
Comment (pyphilia) [inline src/langs/en.json:28]:
```suggestion
  "API_UNAVAILABLE_EXPLANATION": "The Graasp server seems unreachable for the moment.",
```

File: src/langs/en.json [hunk 3]
```diff
@@ -23,5 +23,9 @@
...
+  "API_UNAVAILABLE_INSTRUCTIONS": "Please try again later, or when this message disappears",
```
Comment (pyphilia) [inline src/langs/en.json:29]:
```suggestion
  "API_UNAVAILABLE_INSTRUCTIONS": "Please try again later.",
```

**Review by pyphilia — CHANGES_REQUESTED** [review]

File: src/components/APIChecker.tsx (hunk 1 above)
Comment (spaenleh) [reply src/components/APIChecker.tsx:44]: Okay, i will remove it then.

**Review by pyphilia — APPROVED** [review]
🦜

**Pull request merged**